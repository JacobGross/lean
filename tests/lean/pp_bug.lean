variables p q : Prop
check ¬(p → q)
check ¬p → q
check (¬p) → q
check ¬(p → q)
check p ↔ q
check ¬(p ↔ q)
check ¬p ↔ q
check (¬p) ↔ q
check (p → q) ↔ q
check p → (q ↔ q)
