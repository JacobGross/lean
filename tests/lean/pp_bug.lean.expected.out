¬(p → q) : Prop
¬p → q : Prop
¬p → q : Prop
¬(p → q) : Prop
p ↔ q : Prop
¬(p ↔ q) : Prop
¬p ↔ q : Prop
¬p ↔ q : Prop
p → q ↔ q : Prop
p → (q ↔ q) : Prop
